cmds/tcz/tcz.go

```diff
@@ -87,19 +88,26 @@ func clonetree(tree string) error {
 			return nil
 		}
 		// all else gets a symlink.
-		if link, err := os.Readlink(path); err == nil {
+
+		// If the link exists
+		if link, err := os.Readlink(path[lt:]); err == nil {
+			// Confirm that it points to the same path to be assigned
 			if link == path {
 				return nil
 			}
+
+			// If it does not, return error because tcz packages are inconsistent
 			l.Printf("Symlink: need %v -> %v but %v -> %v is already there", path, path[lt:], path, link)
```

Comment:
I don't think this line is right. Shouldn't `path` and `path[lt:]` here be switched? Based on the code here, `path[lt:]` -> `link`.